zcash_test_vectors/transaction.py

```diff
@@ -549,12 +588,34 @@ def __bytes__(self):
 
         return ret
 
+class TransactionV6(TransactionV5):
+    def init_header(self, consensus_branch_id, rand):
+        # Common Transaction Fields
+        self.nVersionGroupId = V6_VERSION_GROUP_ID
+        self.nConsensusBranchId = consensus_branch_id
+        self.nLockTime = rand.u32()
+        self.nExpiryHeight = rand.u32() % TX_EXPIRY_HEIGHT_THRESHOLD
+        self.zip233Amount = rand.u64() % (MAX_MONEY + 1)
```

Comment:
This results in there being a negligible chance of zero burn, which is unrealistic.
```suggestion
        self.zip233Amount = rand.u64() % (MAX_MONEY + 1)
        if rand.u8() % 2 == 0:
            self.zip233Amount = 0
```